cmd/install/addon/cmd.go

```diff
@@ -197,17 +197,25 @@ func run(cmd *cobra.Command, argv []string) {
 		parameters.Each(func(param *cmv1.AddOnParameter) bool {
 			var val string
 			var hasVal bool
+			input := interactive.Input{
+				Question: param.Name(),
+				Help:     fmt.Sprintf("%s: %s", param.ID(), param.Description()),
+				Required: param.Required(),
+			}
 			// If value is already set in the CLI, ignore interactive prompt
 			flag := cmd.Flags().Lookup(param.ID())
 			if flag != nil {
 				val = flag.Value.String()
 				hasVal = true
 			} else if interactive.Enabled() {
-				input := interactive.Input{
-					Question: param.Name(),
-					Help:     fmt.Sprintf("%s: %s", param.ID(), param.Description()),
-					Required: param.Required(),
+				options, ok := param.GetOptions()
+				if ok {
```

Comment:
Should we use a `interactive.GetOption` prompt when there are predefined options for the parameter?